tests/meta-tag-tests.js

```diff
@@ -0,0 +1,208 @@
+/* eslint no-unused-expressions: "off" */
+const chai = require('chai');
+const yaml = require('js-yaml');
+const fs = require('fs');
+const path = require('path');
+const childProcess = require('child_process');
+const jsdom = require('jsdom');
+
+const { JSDOM } = jsdom;
+const {expect} = chai;
+
+const REDIRECTS_FILE = path.resolve(__dirname, '..', '_data', 'redirect_bases.yaml');
+const redirectsData = fs.readFileSync(REDIRECTS_FILE, 'utf8');
+const redirects = yaml.load(redirectsData);
+const GUIDE_NAMES = fs
+  .readdirSync(path.resolve(__dirname, '..', 'content'), { withFileTypes: true })
+  .filter((entry) => entry.isDirectory())
+  .map((dirEntry) => dirEntry.name);
+const REDIRECTED_GUIDE_NAMES = Object.keys(redirects);
+const REPLATFORMED_GUIDE_NAMES = GUIDE_NAMES.filter(
+  (name) => !REDIRECTED_GUIDE_NAMES.includes(name),
+);
+
+const runEleventy = async (envName = '') => {
+  childProcess.execSync('npx @11ty/eleventy', {
+    cwd: path.resolve(__dirname, '..'),
+    timeout: 15000,
```

Comment:
Suggestion: Should this timeout be set using the same value you are passing into the mocha command in `package.json` ?

Reply:
Good call, addressed here: https://github.com/18F/guides/pull/132/commits/8f4f99d37868d6712623a5d71f9ff1af9fd4232d